chunky/src/java/se/llbit/chunky/world/model/CubeModel.java

```diff
@@ -82,47 +84,53 @@ public CubeModel(Collection<Cube> cubes, double uvScale, Map<String, Texture> te
                 new int[] { 1, 1 },
                 new int[][] { { 0, 0 }, { 0, 1 } },
                 new int[][] { { 2, 1 }, { 2, 0 } },
-                uv);
+                uv,
+                Math.abs(cube.start.y - cube.end.y) > Ray.EPSILON);
```

Comment:
We want double-sided faces if the height is > 0 to e.g. see a helmet from inside the helmet. Same for the other directions. Using single-sided faces for height = 0 allows using the `CubeModel` for entities that have such elements, e.g. the book or the campfire.